test/acceptance/tests/mesh-gateway/mesh_gateway_test.go

```diff
@@ -74,6 +74,8 @@ func TestMeshGatewayDefault(t *testing.T) {
 		"server.extraVolumes[0].load":          "true",
 		"server.extraVolumes[0].items[0].key":  "serverConfigJSON",
 		"server.extraVolumes[0].items[0].path": "config.json",
+		"server.enterpriseLicense.secretName":  "",
+		"server.enterpriseLicense.secretKey":   "",
```

Comment:
We need to do it on the secondary servers since the license should only be applied to the primary.